spring-cloud-starter-stream-sink-cassandra/README.adoc

```diff
@@ -37,3 +53,12 @@ $$cassandra.ttl$$:: $$The time-to-live option of WriteOptions.$$ *($$Integer$$,
 $ mvn clean package
 ```
 
+== Examples
+
+The following example assumes a JSON payload is sent to a default destination called `input`, the sink parses some of its properties (id,time,customer_id,value) and persist them into a table called `test`.
```

Comment:
typo: `persists`.
Typo: called `orders`, not `test`.
See your `insert into` below.

Otherwise LGTM!